src/openforms/config/constants.py

```diff
@@ -62,6 +62,12 @@ class UploadFileType(DjangoChoices):
     heic = ChoiceItem("image/heic", label=(".heic"))
     png = ChoiceItem("image/png", label=_(".png"))
     jpg = ChoiceItem("image/jpeg", label=_(".jpg"))
+    msg = ChoiceItem("application/vnd.ms-outlook", label=_(".msg"))
+    dwg = ChoiceItem(
+        "application/acad.dwg,application/autocad_dwg.dwg,application/dwg.dwg,application/x-acad.dwg,"
+        "application/x-autocad.dwg,application/x-dwg.dwg,drawing/dwg.dwg,image/vnd.dwg,image/x-dwg.dwg",
+        label=_(".dwg"),
+    )
```

Comment:
Originally I tried to group `application` with `application` mime types, but after adding support for other types this doesn't apply any more. So it makes sense to just append them.

Reply:
Or I could arrange the options in alphabetic order. From the user's perspective, that seems to be the most plausible option as it makes scanning the list easier.

Reply:
mm, I think we should stick to the existing grouping and otherwise keep logical grouping, e.g. you want PNG and JPG to be together because they're often allowed together (grouping image types, grouping video types... etc.)